public/docs/ts/latest/guide/forms.jade

```diff
@@ -479,12 +486,11 @@ figure.image-display
     Why "ngForm"? 
     A directive's [exportAs](../api/core/DirectiveMetadata-class.html#!#exportAs) property 
     tells Angular how to link the reference variable to the directive.
-    We set `name` to `ngForm` because the `NgControlName` directive's `exportAs` property happens to be "ngForm".
+    We set `name` to `ngForm` because the `ngModel` directive's `exportAs` property happens to be "ngForm".
+
+    The `exportAs` naming of form directives has been a source of confusion, so very soon, Angular will be renaming the `exportAs` for `ngModel` to 'ngModel'.
     
-    This seems unintuitive at first until we realize that *all* control directives in the 
-    Angular form family &mdash; including `NgForm`, `NgModel`, `NgControlName` and `NgControlGroup` &mdash; *exportAs* "ngForm"
-    and we only ever apply *one* of these directives to an element tag.
-    Consistency rules!
+    This mean we will be able to use the more intuitive `#name=ngModel` when accessing the underlying Angular control.
```

Comment:
I see that this change is already on master, but it doesn't seem to work in rc2. Explained briefly that this change is coming soon though... Removed some of original justification for the same name (consistency) since that goes against the new goal of separate names.